src/bulkExport/getJobStatus.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Handler } from 'aws-lambda';
+import AWS from 'aws-sdk';
+import { BulkExportStateMachineGlobalParameters } from './types';
+import DynamoDbParamBuilder from '../dataServices/dynamoDbParamBuilder';
+import { DynamoDBConverter } from '../dataServices/dynamoDb';
+
+export const getJobStatusHandler: Handler<
+    BulkExportStateMachineGlobalParameters,
+    BulkExportStateMachineGlobalParameters
+> = async event => {
+    const { GLUE_JOB_NAME } = process.env;
+    if (GLUE_JOB_NAME === undefined) {
```

Comment:
Do we want to use falsely here instead of undefined? https://stackoverflow.com/questions/3982663/falsey-values-in-javascript

Reply:
I'm ok with just doing an undefined check here. 

This lambda will fail anyways due to the call to Glue if there's an invalid `GLUE_JOB_NAME` and that's ok. That'd mean that something is wrong on the CFN deployment and a redeployment is needed to fix it.

This is just a convenience to detect if env vars are missing since that's the most likely dev error when assembling the step functions state machine